backend/sozisel/lib/sozisel/video_processor.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Sozisel.VideoProcessor do
+  use FFmpex.Options
+
+  import FFmpex
+
+  @doc """
+  Expects a path to an mp4 file that will be transofmred via ffmpeg to be
+  streamable via http server.
```

Comment:
add some comment tellling that the file will be streamable via http time ranges as it is the main goal over all